source/_components/sensor.dnsip.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: "DNS IP"
+description: "Instructions on how to integrate a DNS IP sensor into Home Assistant."
+date: 2017-02-25 11:05
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Sensor
+ha_iot_class: "Cloud Polling"
+ha_release: 0.40
+---
+
+
+The `dnsip` sensor will expose an IP address, fetched via DNS resolution, as its value. There are two operational modes:
```

Comment:
Well, the main purpose if the sensor is to get the own public IP. But it should be clear, that it can also be used like a regular `nslookup` (which usually is not capable of retrieving the public IP).
So yes, it's technically not two modes. But to me it feels like it does two different things. Don't know how to better describe it.